src/components/CreateManifestForm/CreateManifestForm.tsx

```diff
@@ -154,20 +86,37 @@ const CreateManifestForm: FC<CreateManifestFormProps> = (props) => {
             isRequired
             fieldId="create-satellite-manifest-form-name"
           >
-            <TextInput
-              isRequired
-              type="text"
-              id="create-satellite-manifest-form-name"
-              value={name}
-              placeholder="Name"
-              onChange={handleNameChange}
-              onBlur={validateName}
+            <Controller
+              name="satelliteManifestName"
+              control={control}
+              defaultValue=""
+              rules={{
+                required: 'Please provide a name for your new manifest',
+                maxLength: {
+                  value: 99,
+                  message: 'Your manifest name must be less than 100 characters'
+                },
+                pattern: {
+                  value: /^[0-9A-Za-z_.-]+$/,
+                  message: `Your manifest name must use 
+                only numbers, letters, underscores, hyphens, and periods.`
+                }
+              }}
+              render={({ field }: any) => (
+                <TextInput
+                  isRequired
+                  type="text"
+                  id="create-satellite-manifest-form-name"
+                  placeholder="Name"
+                  {...field}
+                />
+              )}
             />
           </FormGroup>
           <FormGroup
             label="Type"
-            helperTextInvalid="Please select a version for your new manifest"
-            validated={versionValidated}
+            helperTextInvalid={errors.satelliteManifestType?.message}
```

Comment:
Did you forget to add the icon for this form group:
```tsx
helperTextInvalidIcon={<ExclamationCircleIcon />}
```

Reply:
Ah yes, had deleted that by mistake.  Just added back in.